### ISSUE TITLE
Classifier part

### ISSUE THREAD

**Issue by anonymousz97** [LeiCheng-no/CDIL-CNN#1]
https://github.com/LeiCheng-no/CDIL-CNN/blob/main/figures/cdil.png
![image](https://user-images.githubusercontent.com/33414340/150723198-7fa60221-6464-4764-af82-737dc83fb2a4.png)
Hello , as i see the picture and the code in the demo.py i think the classifier self.classifier(torch.mean(y_conv, dim=2)) is not true. Can you explain that for me? The picture is we first pass it through linear layers and aggregate the result. Thanks

![image](https://user-images.githubusercontent.com/33414340/150723154-c715deb4-2e1b-4850-a369-0bc8a4dbf944.png)

**Comment by LeiCheng-no** [comment]
Hello. As we explained in the paper, the average module and linear classification both are linear operations, so they are exchangeable. We found it will speed up the network if we apply the average first.
![image](https://user-images.githubusercontent.com/72392271/150748628-86b3d187-da37-4b20-91e5-5fd02a8002f3.png)
